app/src/main/res/values-sr/strings.xml

```diff
@@ -0,0 +1,387 @@
+    <string name="message_camera_not_permitted">Nema kamere = Nema QR skeniranja!</string>
+
+    <string name="label_copy_viewkey">Vidna šifra</string>
+    <string name="label_copy_address">Javna adresa</string>
+    <string name="label_copy_xmrtokey">XMR.TO šifra</string>
+    <string name="message_copy_viewkey">Vidna šifra kopirana u klipbord!</string>
+    <string name="message_copy_xmrtokey">XMR.TO šifra kopirana u klipbord!</string>
+    <string name="message_copy_address">Adresa novčanika kopirna u klipbord!</string>
+    <string name="message_copy_txid">ID transakcije kopiran u klipord!</string>
+    <string name="message_nocopy">Kopiranje je onemogućeno iz sigurnosnih razloga!</string>
+
+    <string name="message_exchange_failed">Ne može se dobiti kurs!\nUse XMR/XMR or try again</string>
+
+    <string name="generate_title">Kreiraj novčanik</string>
+    <string name="generate_name_hint">Ime novčanika</string>
+    <string name="generate_password_hint">Fraza-lozinka novčanika</string>
+    <string name="generate_fingerprint_hint">Dozvoli otvaranje novčanika otiskom prsta</string>
+    <string name="generate_fingerprint_warn"><![CDATA[
+        <strong>Autorizacija otiska prsta</strong>
+        <p>Sa omogućenom autorizacijom otiska prsta možeš videti stanje u novčaniku i primiti sredstva
+        bez unosa lozinke.</p>
+        <p>Ali za dodatnu sigurnost, monerujo će ipak tražiti da uneseš lozinku za
+        pregledanje novčaničkih detalja ili slanje sredstava.</p>
+        <strong>Sigurnosno upozorenje</strong>
+        <p>Konačno, monerujo želi da te podseti da biloko ko može imati tvoj otisak prsta biće
+        sposoban da zaviri u stanje tvog novčanika.</p>
+        <p>Na primer, zlonamerni korisnik iz tvoje okoline može ti otvoriti novčanik dok spavaš.</p>
+        <strong>Da li si siguran da omogućavaš ovu funkciju?</strong>
+    ]]></string>
+    <string name="generate_bad_passwordB">Fraze-lozinke se ne podudaraju</string>
+    <string name="generate_empty_passwordB">Fraza-lozinka ne može biti prazna</string>
+    <string name="generate_buttonGenerate">Napravi mi već novčanik!</string>
+    <string name="generate_button_accept">Primetio sam mnemoničko seme</string>
+
+    <string name="generate_wallet_name">Daj mi ime!</string>
+    <string name="generate_wallet_exists">Novčanik postoji!</string>
+    <string name="generate_wallet_dot">Nemožeš početi sa .</string>
+    <string name="generate_wallet_creating">Kreiranje novčanika</string>
+    <string name="generate_wallet_created">Novčanik kreiran</string>
+
+    <string name="generate_restoreheight_error">Unesi broj datuma u formatu (YYYY-MM-DD)</string>
+
+    <string name="generate_wallet_type_key">Šifre</string>
+    <string name="generate_wallet_type_new">Novo</string>
+    <string name="generate_wallet_type_seed">Seme</string>
+    <string name="generate_wallet_type_view">Pregled</string>
+    <string name="generate_wallet_type_ledger" 
```

Comment:
This whole string need to be removed. As I was saying in `#monero-translatons`. If you want to you can add me as collaborator to your repository, so that i can fix the technical issues for you.

Reply:
Oh, I miss that. :( How can I add you as a collaborator?

Reply:
Don't worry. No big deal.
Go to your forked repository ->  collaborators & team -> collaborators

Reply:
BTW In this case, you  can easily delete the string on github using the 'edit' button

Reply:
Just in case, I added you as a collaborator. I don't want to make further mistakes. :) Thank you again erciccione.

Reply:
Perfect. I'm gonna fix this string now. If further problems appear, i will fix those as well :)

Reply:
Thanks man. You are a hero!